### ISSUE TITLE
perlbrew pulling cperl

### ISSUE THREAD

**Issue by sigzero** [gugod/App-perlbrew#562]
I just ran "perlbrew available" on macOS. It reported the following as one of the options:

cperl-5.26.1      URL: <https://github.com/perl11/cperl/archive/cperl-5.26.1.tar.gz>

Is perlbrew now doing non-standard Perl too? I don't think it should deviate into that territory.

**Comment by JJ** [comment]
Besides, it does not actually work:
```
$ perlbrew install cperl-5.27.1
Can't locate object method "release_detail_cperl_remote" via package "App::perlbrew" at /home/jmerelo/perl5/perlbrew/bin/perlbrew line 937.
```

**Comment by gugod** [comment]
I somehow wrongly revert that subroutine in commit aa42ab80d9ec1998d3a955e118b54986abf9f923

my bad :)

**Comment by gugod** [comment]
@rubiot It is not pushed to github yet, but I intend to fix this in the next release.

**Comment by rubiot** [comment]
OK. Thanks.

**Comment by gugod** [comment]
This is fixed in the fresh release 0.81.`perlbrew install cperl-5.27.2` should just work -- at least it should start compiling. :)
